dev-client/src/theme.ts

```diff
@@ -78,7 +78,7 @@ export const theme = extendTheme({
       primary: '#1A202C',
       secondary: '#33474F',
       disabled: '#9EA8AB',
-      locationicon: '#001923B3',
+      icon: '#001923B3',
```

Comment:
Huh, weird of me to have used alpha here; I cannot recall if/why that was an intentional design choice -- do we want this to be fully opaque?

Reply:
I think this is verbatim from the design file.